src/dialects/yugabyte/data-types.js

```diff
@@ -0,0 +1,597 @@
+'use strict';
```

Comment:
@priyanshi-yb are doing we any changes in data-types.js? I remember last time we discussed to just use same as that of postgres

Reply:
We require to map the datatypes for yugabytedb dialect and there are changes in this file with postgres by mapping these types for yugabytedb because as I had mentioned earlier that there is some parser check for each data-types in connection manager with the current dialect name. So, we have to keep the mapping specific to yugabytedb dialect.

Reply:
ok got it.